Description: Use a sensible value for PATH_MAX if it is not defined.
Author: Peter Pentchev <roam@ringlet.net>
Forwarded: no
Last-Update: 2017-01-12

--- a/include/osdep.h
+++ b/include/osdep.h
@@ -480,6 +480,10 @@
 #define ETHER_ADDR_LEN  (6)
 #endif /* !ETHER_ADDR_LEN */
 
+#ifndef PATH_MAX
+#define PATH_MAX	1024
+#endif
+
 #if !HAVE_STRUCT_IPOPTS
 #define   MAX_IPOPTLEN   40
 struct ipoption {
